media/css/firefox/new/compare.scss

```diff
@@ -467,7 +467,7 @@ $image-bottom-height: 28px;
     .c-compare-table {
         th,
         td {
-            padding: $spacing-sm;
+            padding: $spacing-sm $spacing-lg;
```

Comment:
The extra long text in this PR revealed an issue with the highlight-shadow pseudo elements, whereby the shadow could obscure the cell contents at various screen sizes. I've bumped up the default padding here to a size larger than the width of the pseudo elements to prevent this happening.